src/shell/components/global-notifications/GlobalNotifications.js

```diff
@@ -72,95 +126,155 @@ export default connect((state) => {
-                    key={i}
-                    className={cx(styles.Notification, styles.bodyText)}
-                  >
-                    <p className={styles.Message}>
-                      <FontAwesomeIcon
-                        className={cx(
-                          notice.kind ? styles[notice.kind] : null,
-                          styles.Icon
-                        )}
-                        icon={msgIcon(notice.kind)}
-                      />
-                      {notice.message}
-                    </p>
-                    {notice.epoch && (
-                      <small className={cx(styles.Timestamp, styles.caption)}>
-                        {moment(notice.epoch).fromNow()}
-                      </small>
-                    )}
-                  </li>
-                );
-              })}
-            </ul>
-          </DrawerContent>
-        </Drawer>
-      </aside>
+
+              <ul>
+                {props.notifications.map((notice, i) => {
+                  return (
+                    <li
+                      key={i}
+                      className={cx(styles.Notification, styles.bodyText)}
+                    >
+                      <p className={styles.Message}>
+                        <FontAwesomeIcon
+                          className={cx(
+                            notice.kind ? styles[notice.kind] : null,
+                            styles.Icon
+                          )}
+                          icon={msgIcon(notice.kind)}
+                        />
+                        {notice.message}
+                      </p>
+                      {notice.epoch && (
+                        <small className={cx(styles.Timestamp, styles.caption)}>
+                          {moment(notice.epoch).fromNow()}
+                        </small>
+                      )}
+                    </li>
+                  );
+                })}
+              </ul>
+            </DrawerContent>
+          </Drawer>
+        </aside>
+      </SnackbarProvider>
     );
   })
 );
+
+export const CustomNotification = forwardRef(({ id, ...props }, ref) => {
+  const { closeSnackbar } = useSnackbar();
+
+  const handleDismiss = useCallback(() => {
+    closeSnackbar(id);
+  }, [id, closeSnackbar]);
+
+  return (
+    <SnackbarContent ref={ref}>
+      <Alert
+        key={id}
+        variant="filled"
+        severity={props.severity}
+        icon={props.icon}
+        action={
+          <Stack direction="row">
+            {/* if there's additional action show this */}
+            {/* <Button sx={{ color: "white" }}>Action</Button> */}
+            <IconButton onClick={handleDismiss}>
+              <CloseIcon sx={{ width: 20, height: 20, color: "white" }} />
+            </IconButton>
+          </Stack>
+        }
+        sx={{
+          "& .MuiAlert-icon, .MuiAlert-message, .MuiAlert-action ": {
+            color: "#fff",
+          },
+          "& .MuiAlert-action": {
```

Comment:
Can we do these changes one off for this component for now? Just thinking if this will affect on other part of website that are using Alert Component..